_includes/addon_table.html

```diff
@@ -99,9 +99,13 @@
                 var url = $(row).find('.name').data('url');
 
                 // Assign click event to the entire row
-                $(row).click(function() {
-                    if (url) {
-                        window.location.href = url;
+                $(row).on('mousedown', function(e) {
+                    if (e.target.tagName.toLowerCase() !== 'a' && url) {
+                        if(e.which === 2 || e.ctrlKey || e.metaKey) {
+                            window.open(url);
+                        } else {
```

Comment:
We want the usual action to happen with a left-click only.
It feels odd that a right-click both opens the add-on page and shows the context menu.

```suggestion
                        } else if (e.which === 1) {
```